geolocator_android/lib/src/types/foreground_settings.dart

```diff
@@ -0,0 +1,74 @@
+/// Uniquely identifies an Android resource
+class AndroidResource {
+  /// The name of the desired resource.
+  final String name;
+
+  /// Optional default resource type to find, if "type/" is not included in the name. Can be null to require an explicit type.
+  final String defType;
+
+  /// Uniquely identifies an Android resource
```

Comment:
Nit: sentence should end with a period.
```suggestion
  /// Uniquely identifies an Android resource.
```